app/services/import_mitigation.rb

```diff
@@ -0,0 +1,52 @@
+class ImportMitigation
+  # rubocop:disable LineLength
+  MITIGATION_CATEGORIES_FILEPATH = 'public/mocks/mitigation_categories.json'.freeze
+  MITIGATION_ACTIONS_FILEPATH = "#{CW_FILES_PREFIX}3. Mitigation/Table 27 - Additional mitigation actions in energy sector not in BUR1 - Sheet1.csv".freeze
```

Comment:
I think there are more files with data for "mitigation actions", not just this one - see the blue text in the data review file. What would make a lot of sense if all that data is effectively landing in the same table in "Mitigation Actions" is to ask for those files to be merged, so that they don't need separate importers. I could help to suggest the exact file format to be used across all those files.